src/createDelete.js

```diff
@@ -0,0 +1,61 @@
+import dots from './assets/dots.png';
+import deleteIcon from './assets/delete.png';
+import { tasks } from './addTask.js';
+import * as storage from './storageFunc.js';
+
+const list = document.querySelector('.list');
+
+const createTask = () => {
+  list.innerHTML = '';
+  tasks.forEach((task, index) => {
+    const li = document.createElement('li');
+    li.classList.add('list-item');
+    li.innerHTML = `
+      <input type="checkbox" value="${task.completed}">
+      <p class="li-text">${task.description}</p>
+      <img src="${deleteIcon}" id="${index}" class="delete-icon">
+      <img src="${dots}" id="${index}" class="dots-img">
+    `;
+    list.appendChild(li);
+    task.index = index.toString();
+
```

Comment:
- [ ] The index of the tasks is supposed to be integer. And it's required that the task index of the last task should be equal to the total number of tasks in the tasks list. Kindly work on arranging the tasks index to match the total number of tasks in the list and also when a task is deleted, the total count of tasks must match the list. 👍🏽

<img width="1136" alt="Screenshot 2023-05-12 at 7 35 49 PM" src="https://github.com/shm04/to-do-list/assets/50941074/58b0899e-5e62-4f15-ab26-bd0007e05f7b">